airflow/providers/google/cloud/operators/automl.py

```diff
@@ -338,6 +339,11 @@ def execute(self, context: Context):
         return PredictResponse.to_dict(result)
 
 
+@deprecated(
+    planned_removal_date="January 01, 2025",
+    use_instead="airflow.providers.google.cloud.operators.vertex_ai.batch_prediction_job",
+    category=AirflowProviderDeprecationWarning,
+    )
```

Comment:
Could you please add a test for the deprecation? (`with pytest.warns(...)`)

Reply:
Thanks for mentioning it,  I updated the tests affected.